test-it/ViaTests/ClusterBinaryOpBug.via

```diff
@@ -0,0 +1,55 @@
+define(clusterBinaryOpBug dv(.VirtualInstrument (
```

Comment:
Please add a similar test for the following cases:

- Array of clusters
- Unary Op (for both the cluster case you have and the array of clusters case)

 array of clusters.